tests/conftest.py

```diff
@@ -5,6 +5,66 @@
 pytest_plugins = ["pytest_dallinger"]
 
 
+@pytest.fixture(scope="module")
+def check_firefox(request):
```

Comment:
Fixtures like this the currently recommended/required way to skip tests, I'm guessing?

Reply:
The config is not accessible anymore from the pytest module, which requires fixtures like this one.